.github/workflows/api_reference.yaml

```diff
@@ -16,17 +16,17 @@ jobs:
         with:
           path: src/Fast-DDS-QoS-Profiles-Manager
 
-      - name: Install doxygen and cmake
-        run: |
-          sudo apt install doxygen cmake
+      - name: Install apt packages
+        uses: eProsima/eProsima-CI/ubuntu/install_apt_packages@main
+        with:
+          custom_packages: doxygen cmake
 
       - name: Build library
         run: |
           cd src/Fast-DDS-QoS-Profiles-Manager/lib
           mkdir build && cd build
           cmake ..
-          make
-          cmake --install . --prefix "../install"
+          make && cmake --install . --prefix "../install"
```

Comment:
Could we try and use CMake directly? (I know that the README currently gives these steps...). Another option is to use colcon directly and use the generic action in the same PR already mentioned.